app/services/foreman_ansible/ansible_report_scanner.rb

```diff
@@ -15,8 +15,7 @@ def scan(report, logs)
       def ansible_report?(logs)
         return false if logs.blank?
         logs.any? do |log|
-          log['log'].fetch('messages', {}).
-            fetch('message', '') =~ /"_ansible_parsed"/
+          log['log'].fetch('reporter', nil) == 'ansible'
```

Comment:
Isn't `a_hash.fetch('reporter', nil)` essentialy equivalent to `a_hash['reporter']`?

Reply:
Yeah, updated.